src/cloud/project/new-relic.md

```diff
@@ -18,11 +26,32 @@ functional_areas:
 
 Your {{site.data.var.ece}} account includes the software for the New Relic APM service along with a license key. You do not need to purchase or install any additional software.
 
-On {{site.data.var.ece}} Pro accounts, the Production environment also includes the [New Relic Infrastructure (NRI)][New Relic infrastructure] service, which automatically connects with the application data and performance analytics to provide dynamic server monitoring.
+## New Relic Infrastructure
+
+{{site.data.var.ece}} Pro projects include the [New Relic Infrastructure (NRI)][New Relic infrastructure] service, which automatically connects with the application data and performance analytics to provide dynamic server monitoring. This service is available only on Production environments.
 
 {:.bs-callout-info}
 For Pro accounts, if New Relic APM is not installed on the Staging and Production environments or New Relic Infrastructure is not available in the Production environment, submit a [support ticket]({{ site.baseurl }}/cloud/trouble/trouble.html) to request installation.
 
+## New Relic Logs
+
+{{site.data.var.ece}} Pro projects include the [New Relic Logs][] service on both Production and Staging environments. The service is pre-configured to aggregate all log data from your Staging and Production environments and display it in a centralized log management dashboard.
+
+The aggregated data includes information from the following logs:
+
+-  All {{site.data.var.ct}} and Magento application logs from the `~/var/log` directory
+-  Logs for cloud services from the `var/log/platform/<project ID>` directory
```

Comment:
This is a small nit. I try to avoid using space in variables. I think it looks better and editor apps don't fuss over it as much.

```suggestion
-  Logs for cloud services from the `var/log/platform/<project-ID>` directory
```

Reply:
Should be `/var/log/platform/` etc